cardinal/system_test.go

```diff
@@ -83,3 +87,9 @@ func TestCanRegisterMultipleSystem(t *testing.T) {
 	assert.Check(t, firstSystemCalled)
 	assert.Check(t, secondSystemCalled)
 }
+
+func TestCannotRegisterSameSystemTwice(t *testing.T) {
```

Comment:
Add a test that inserts 2 systems with distinct names and assert that no error is generated (I predict that new test will fail).

Reply:
resolved ser